### PR TITLE
Add variables for land cover

### PR DESCRIPTION
**Pull request by danielhuppmann**
This PR adds variables for land cover based on the xlsx overview developed by @flohump @realxinzhao and @kanishkan91.

This PR does NOT include subcategories for cropland and forests - this will be implemented in a follow-up PR once this discussion is finished and the iniatl set of land-cover variables are merged.

A few observations:
- I added the variable "Land Cover|Water Ecosystems", which has been missing from the templates in NAVIGATE and ENGAGE
- I assigned all water-ecosystems variables as tier 3
- Is there a particular reason why glaciers are a sub-category of water ecosystems?
- The variable for built-up area land cover was renamed for consistency with the naming convention, see [here](https://docs.ece.iiasa.ac.at/iamc/variable.html#variable-naming-conventions)

Please review the descriptions.

@IAMconsortium/common-definitions-land

### CODE REVIEW

File: definitions/variable/land/land-cover.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    definition: Pasture, not only high quality rangeland, based on FAO definition of
+      "permanent meadows and pastures"
+    unit: million ha
+    tier: 2
+- Land Cover|Cropland:
+    definition: Arable land, i.e. land in bioenergy crop, food, and feed/fodder crops,
+      permanent crops as well as other arable land (physical area)
+    unit: million ha
+    tier: 2
+    agmip: Land Cover (LAND) - All crops (CRP)
```
Comment (flohump) [inline definitions/variable/land/land-cover.yaml:22]:
```suggestion
    agmip: Land Cover (LAND) - All crops (CRP)
- Land Cover|Cropland|Cereals:
    definition: Cropland with maize, rice, temperate cereals and tropical cereals (physical area)
    unit: million ha
    tier: 3
    agmip: 
- Land Cover|Cropland|Oil Crops:
    definition: Cropland with cotton seed, groundnuts, oilpalms, other oil crops incl rapeseed, soybean, sunflower (physical area)
    unit: million ha
    tier: 3
    agmip: 
- Land Cover|Cropland|Sugar Crops:
    definition: Cropland with sugar beet, sugar cane (physical area)
    unit: million ha
    tier: 3
    agmip: 
- Land Cover|Cropland|Other Crops:
    definition: Cropland with fruits, vegetables, nuts, potatoes, pulses, tropical roots (physical area)
    unit: million ha
    tier: 3
    agmip: 
- Land Cover|Cropland|Energy Crops:
    definition: Cropland with short rotation grasses, short rotation trees (physical area)
    unit: million ha
    tier: 3
    agmip: 
```

File: definitions/variable/land/land-cover.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- Land Cover|Forest:
+    definition: Managed and unmanaged forest area
+    unit: million ha
+    tier: 2
+    agmip: Land Cover (LAND) - Forest (FOR)
+
```
Comment (flohump) [inline definitions/variable/land/land-cover.yaml:28]:
```suggestion
- Land Cover|Forest|Managed:
    definition: Managed forest area incl. timber plantations and re-/afforestation areas
    unit: million ha
    tier: 2
    agmip: 
- Land Cover|Forest|Managed|Timber Plantations:
    definition: Managed and unmanaged forest area
    unit: million ha
    tier: 2
    agmip: Land Cover (LAND) - Forest (FOR)
- Land Cover|Forest|Managed|Afforestation
    definition: Afforestation area
    unit: million ha
    tier: 2
    agmip: 
- Land Cover|Forest|Managed|Reforestation
    definition: Reforestation area
    unit: million ha
    tier: 2
    agmip: 
- Land Cover|Forest|Secondary Forest:
    definition: Secondary forest area
    unit: million ha
    tier: 2
    agmip: 
- Land Cover|Forest|Primary Forest:
    definition: Primary forest area
    unit: million ha
    tier: 2
    agmip: 
```

File: definitions/variable/land/land-cover.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- Land Cover|Water Ecosystems:
```
Comment (flohump) [inline definitions/variable/land/land-cover.yaml:29]: I suggest to move Water Ecosystems to seperate file and remove the prefix "Land Cover" like `Water Ecosystems|Forests`
We need peatlands as a subtype of wetlands
Water Ecosystems|Wetlands|Peatlands
Water Ecosystems|Wetlands|Peatlands|Intact
Water Ecosystems|Wetlands|Peatlands|Drained
Water Ecosystems|Wetlands|Peatlands|Rewetted

**Review by flohump — COMMENTED** [review]
I made some suggestions but also have a question. See comment above.

**Comment by danielhuppmann** [comment]
Removed the water-ecosystems land cover variables

File: definitions/variable/land/land-cover.yaml (hunk 3 above)
Comment (danielhuppmann) [reply definitions/variable/land/land-cover.yaml]: Removed

**Comment by danielhuppmann** [comment]
Thanks for the extensions @flohump - merging now, let's revisit extensions and clarifications in follow-up PRs.

**Pull request merged**